WISE-Community/WISE-Client
# feat(Export): Create export student work route

## Changes
- Created "Export Student Work" route that replaces "Export Latest Work" and "Export All Work"
- "Export Student Work" lets you choose between "Latest" and "All"
- Cleaned up DataExportComponent by removing code related to "Export Latest Work" and "Export All Work"

## Test
- Make sure "Export Student Work" for "Latest" and "All" still works properly
- Make sure the other exports generated directly from DataExportComponent still work such as
  - Export Latest Notebook Items
  - Export All Notebook Items
  - Export Notifications
  - Export Student Assets

Closes #1763
